### PR TITLE
 Use a gloabl flag to enable/disable serial logging

### PR DESCRIPTION
**Pull request by Silverlined**
In relation to one of the opened issues - 
Use a gloabl flag to enable/disable debug output, I simply added a few lines of code to address it.

This is one of the first times I make pull request and I hope I do it right. Thank you for the cool library!

### CODE REVIEW

File: InfluxDb.cpp [hunk 1]
```diff
@@ -76,24 +84,31 @@ boolean Influxdb::write(InfluxData data) { return write(data.toString()); }
  * for a list of error codes.
  */
 boolean Influxdb::write(String data) {
-  Serial.print(" --> writing to " + _db + ":\n");
-  Serial.println(data);
+  if (_debug == true) {
```
Comment (tobiasschuerg) [inline InfluxDb.cpp]: Just `if (_debug)` would be enough

File: InfluxDb.cpp [hunk 2]
```diff
@@ -76,24 +84,31 @@ boolean Influxdb::write(InfluxData data) { return write(data.toString()); }
...
+    Serial.print(" --> writing to " + _db + ":\n");
+    Serial.println(data);
 
-  int httpResponseCode = http.POST(data);
-  Serial.print(" <-- Response: ");
-  Serial.print(httpResponseCode);
+    int httpResponseCode = http.POST(data);
```
Comment (tobiasschuerg) [inline InfluxDb.cpp]: This part need to be outside of `if-debug` - otherwise the post will only be executed of debug is enabled.

**Comment by tobiasschuerg** [comment]
If you mention #15 then the issue will automatically be linked.

File: InfluxDb.cpp (hunk 2 above)
Comment (Silverlined) [reply InfluxDb.cpp]: True, that is why I have included it into the `else` (starting at line 104) part of `if(_debug)` as well. It is pretty much duplicated without the Serial statements, I know that is not the most elegant solution but I could not decide on a better way.

File: InfluxDb.cpp (hunk 1 above)
Comment (Silverlined) [reply InfluxDb.cpp]: Updated.

File: InfluxDb.cpp (hunk 2 above)
Comment (tobiasschuerg) [reply InfluxDb.cpp]: I think we should not duplicate the logic but only wrap logging in an if statement.

**Pull request closed without merging**